package.json

```diff
@@ -13,7 +13,10 @@
     "dist-tags": "npm run exec -- npm dist-tag ls",
     "install-packages": "npm run exec -- npm install & npm install",
     "exec": "lerna exec --concurrency 1 --no-bail --no-sort --stream",
+    "lint": "lerna run lint --parallel",
```

Comment:
Does --parallel give a good experience? For the "exec" script I set concurrency to 1 because otherwise the output was jumbled between different packages.

Reply:
It's fine to try it this way, it can always be easily changed.

Reply:
The output is jumbled . . . but it's way faster.